File: src/User/DTOs/GetUserDTO.cs
namespace WebApplication1.User.DTOs;

public class GetUserDto
{
    public int Id { get; set; }
    public string FirstName { get; set; } = null!;
    public string LastName { get; set; } = null!;
    public string Email { get; set; } = null!;
    public string PhoneNumber { get; set; } = null!;
    public string Role { get; set; } = null!;
    public string Status { get; set; } = null!;
    public int? CompanyId { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}

File: frontend/src/dtos/getUserDTO.ts
import {ErrorDTO} from "@/src/utils/dtos/errorDTO";

export type GetUserDTO = {
    Id: number
    firstName: string;
    lastName: string;
    email: string;
    phoneNumber: string;
    role: string;
    status: string;
    companyId: number;
    createdAt: Date | undefined;
    updatedAt: Date | undefined;
    error?: ErrorDTO;
}